src/shutdown.py

```diff
@@ -1,20 +1,25 @@
+"""
+src/shutdown.py
+===============
+"""
 import os
 import Queue
 import threading
 import time
 
+import shared
+import state
 from debug import logger
 from helper_sql import sqlQuery, sqlStoredProcedure
 from helper_threading import StoppableThread
-from knownnodes import saveKnownNodes
 from inventory import Inventory
+from knownnodes import saveKnownNodes
 from queues import (
     addressGeneratorQueue, objectProcessorQueue, UISignalQueue, workerQueue)
-import shared
-import state
 
 
 def doCleanShutdown():
+    """Used to tell proof of work worker threads and the objectProcessorThread to exit."""
     # Used to tell proof of work worker threads
```

Comment:
you can remove line 23 and 24

Reply:
Oh, yes. I totally forget to remove that comment.  Will remove the code asap.
Thank you.